main.go

```diff
@@ -159,19 +157,10 @@ func completer(d prompt.Document) []prompt.Suggest {
 		}
 
 		if command == "pull" {
-
-			if len(strings.Split(d.Text, " ")) > 2 {
-				return []prompt.Suggest{}
-			}
-
-			if len(word) < 3 {
-				return []prompt.Suggest{}
-			}
-
-			if word != command {
+			if word == "" || len(word) > 2 {
```

Comment:
The removed control `if len(strings.Split(d.Text, " ")) > 2` was preventing image suggestion after already select an image. This changes breaks it.

Reply:
You're right, it's my mistake. I've fixed it.